src/bioregistry/data/bioregistry.json

```diff
@@ -89781,7 +89781,7 @@
       "synonyms": [
         "poc"
       ],
-      "uri_format": "http://www.plantontology.org/amigo/go.cgi?view=details&search_constraint=terms&depth=0&query=$1"
+      "uri_format": "https://browser.planteome.org/amigo/term/PO:$1"
```

Comment:
Similar to the above, this is a URL under `prefixcommons` and not something we would edit here.